docs/concepts/cluster_queue.md

```diff
@@ -0,0 +1,24 @@
+# Cluster Queue
+
+A `ClusterQueue` is a cluster-scoped resource that governs a pool of resources
+such as CPU, memory and hardware accelerators. A `ClusterQueue` defines:
+- The resource _flavors_ that it manages, with usage limits and order of consumption.
+- Fair sharing rules across the tenants of the cluster.
+
+Only cluster administrators should create `ClusterQueue` objects.
+
+## Resource Flavors
+
+Resources in a cluster are typically not homogeneus. Resources could differ in:
+- pricing and availability (ex: spot vs on-demand VMs)
+- architecture (ex: x86 vs ARM CPUs)
+- brands and models (ex: Radeon 7000 vs Nvidia A100 vs T4 GPUs)
+
+A `ResourceFlavor` is a resource that represents these variations and allows
```

Comment:
```suggestion
A `ResourceFlavor` is an object API that represents these variations and allows
```

just so it is not confused with compute resource

Reply:
Done, updated everywhere.